modules/power-monitoring-deploying-kepler-redfish.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/installing-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-deploying-kepler-redfish_{context}"]
+= Deploying {PM-kepler} using Redfish API
```

Comment:
Since Redfish API isn't always available, how can you check that it's available?

Reply:
This information has to be known to the cluster admin. Kepler does not have a way to detect if redfish is supported on the node.

Reply:
Is there a command you can run to get this information?